app/tests/fixtures/credentials.py

```diff
@@ -433,3 +433,79 @@ async def get_or_issue_regression_cred_valid(
         referent=valid_credential["referent"],
         cred_def_revocable=valid_credential["cred_def_id"],
     )
+
+
+@pytest.fixture(scope="function")
+async def issue_alice_creds_non_revoke(
+    faber_client: RichAsyncClient,
+    alice_member_client: RichAsyncClient,
+    credential_definition_id: str,
+    faber_and_alice_connection: FaberAliceConnect,
+) -> List[CredentialExchange]:
+
+    faber_conn_id = faber_and_alice_connection.faber_connection_id
+
+    faber_cred_ex_ids = []
+    for i in range(10):
```

Comment:
can we parameterise the number of creds somehow? Ideally this should be configurable by the test calling the fixture. GPT should know but I think there's a way to pass params to fixtures from the test using it.

For our test we don't need 10 - that'll slow down things. Just 3 should be enough.